packages/qdrant-loader/src/qdrant_loader/core/worker/queue.py

```
@@ -0,0 +1,218 @@
+            visibility_deadline=None,
+        )
+
+        async with self._session_factory() as session:
+            session.add(job)
+            await session.commit()
+            await session.refresh(job)
+            return job
+
+    async def claim_next(self, lease_seconds: int = 60) -> Job | None:
+        if lease_seconds < 0:
+            raise ValueError("lease_seconds must be non-negative")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)